httphandler/roundtripper_decorators.go

```diff
@@ -36,8 +36,11 @@ func (lrt *loggingRoundTripper) RoundTrip(req *http.Request) (resp *http.Respons
 		statusCode,
 		duration,
 		errStr)
-
-	lrt.accessLog.Println(accessLogMessage.String())
+	jsonb, almerr := accessLogMessage.JSON()
+	if almerr != nil {
+		log.Println(almerr.Error())
+	}
+	lrt.accessLog.Printf("%s", jsonb)
```

Comment:
It will be always executed, even if `JSON()` method fails - so it will log an empty string?